operatorcourier/build.py

```diff
@@ -3,14 +3,14 @@
 
 class BuildCmd():
     def _get_empty_bundle(self):
-         return dict(
+        return dict(
```

Comment:
There are no actual changes in this file. Please remove this formatting only change as it just adds noise to the commit log.